### PR TITLE
change 'Places' from array -> object to to conform to dictionary structure

### PR DESCRIPTION
**Pull request by BenMacnaughton**
Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [x] Reference the issue this is meant to fix or describe it if none exists.
- [x] Full description of changes made.

## Pull Request Checklist

- [x] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [x] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [x] The Docker image builds
- [x] The container runs correctly when sent inputs with the changes
- [x] No models or other large files are part of these commits
- [ ] A description of the tests already run as part of this PR is present

### CODE REVIEW

**Review by JRegimbal — CHANGES_REQUESTED** [review]
The schema with the changes here doesn't make sense. You have places as type object, which means it should have properties, but instead assign it items, which imply an array. Can you provide a sample of what places should look like in the output?

**Comment by BenMacnaughton** [comment]
Places appears in this format: `"places":{"A19RWFJe":{"addr":"1655 Rue Saint-Denis","cat":12,"from":"foursquareexplore","hours":"    ","ll":[45.515116,-73.563492],"phone":"5144395535","rating":8.2,"title":"Mâche!","website":"http://restaurantmache.com"},"A1lWUVQC":{"addr":"800 boul. De Maisonneuve Est, 17e étage","cat":71,"from":"foursquaresearch","ll":[45.516391999999996,-73.559775],"title":"Arrondissement Ville-Marie","website":"http://ville.montreal.qc.ca/villemarie"}}`

It is an unordered set of key-value pairs with the key being a unique place ID. We thought this would be a good way to track specific places down the line. We could change it to an array in the preprocessor but we felt that this K/V format could make it easier to identify places.

**Comment by BenMacnaughton** [comment]
Two changes have been made: 1) Autour schema updated to reflect its dictionary structure using `additionalProperties` 2) a new maps request option has been created that includes a `placeID` field in place of the coordinates for situations where coordinates were not found on the browser side.

File: preprocessors/autour.schema.json
Comment (JRegimbal) [inline preprocessors/autour.schema.json:38]: This isn't how additionalProperties works...it is either true or false to indicate if you can have additional properties in an object along with those explicitly listed in properties.

File: request.schema.json
Comment (JRegimbal) [inline request.schema.json:42]: What actually is this ID? Where does it come from? Why is it required?

**Review by JRegimbal — CHANGES_REQUESTED** [review]
Schema still doesn't make sense...I'm also not sure why an object containing POIs indexed by some ID key is useful. Is there even a case where someone would know the ID ahead of time? To me, it seems like an array of objects where the object contains an ID field (as it was previously) would be better, though I'm willing to be convinced otherwise.

File: request.schema.json
Comment (aidanwilliams09) [reply request.schema.json:42]: We haven't gotten to the browser side for this yet, but it is the substitute for a lat/long. eg the map may be a query of the Eiffel Tower instead of a lat/long and therefore we would make ```placeID='Eiffel+Tower,Paris+France```

Comment (JRegimbal) [reply request.schema.json:42]: So this is a google specific ID? Or is it just a query?

Comment (BenMacnaughton) [reply request.schema.json:42]: It is called a placeID by Google and can be structured in plain text or by a unique code as I understand

Comment (BenMacnaughton) [reply request.schema.json:42]: The embedded map request allows for an english like query or an ID (https://developers.google.com/maps/documentation/places/web-service/place-id)

**Comment by JRegimbal** [comment]
@BenMacnaughton @aidanwilliams09 what's the status of this?

**Comment by BenMacnaughton** [comment]
@JRegimbal just waiting for approval I believe

**Comment by JRegimbal** [comment]
Alright, you never reassigned so I was waiting on you. Is the autour preprocessor currently producing outputs that match this new schema?

File: request.schema.json
Comment (JRegimbal) [inline request.schema.json:43]:
```suggestion
            "description": "The Google place ID of the location identified in a map.",
```

**Review by JRegimbal — APPROVED** [review]

**Pull request merged**
